templates/grpc.gemspec.template

```diff
@@ -42,7 +42,8 @@
     s.add_development_dependency 'logging',            '~> 2.0'
     s.add_development_dependency 'simplecov',          '~> 0.14.1'
     s.add_development_dependency 'rake',               '~> 13.0'
-    s.add_development_dependency 'rake-compiler',      '~> 1.1'
+    # rake-compiler 1.1.2 and 1.1.3 are broken, see b/209603283
```

Comment:
same comment as in https://github.com/grpc/grpc/pull/28320: can we try pinning this with:

'<= 1.1.1' ?

to fix macos artifacts?